src/integ_test/java/games/strategy/engine/lobby/server/db/MutedMacControllerIntegrationTest.java

```diff
@@ -59,8 +59,12 @@ public void testMuteMacUpdate() {
   }
 
   private Instant muteMacForSeconds(final long length) {
-    final Instant muteEnd = length == Long.MAX_VALUE ? null : Instant.now().plusSeconds(length);
+    final Instant muteEnd = length == Long.MAX_VALUE ? null : nanosIgnored(Instant.now().plusSeconds(length));
     controller.addMutedMac(hashedMac, muteEnd);
     return muteEnd;
   }
+  
+  private Instant nanosIgnored(final Instant precise) {
```

Comment:
I know you said you plan on removing these workarounds in a follow-up PR, but just in case that gets delayed for some reason, a big fat FIXME comment referencing this PR might be a good idea.

If, for some reason, you think these methods may exist for some time, a better name might be `withoutNanos()`.

Nit: These methods can be static.